sql/parser/scan.go

```diff
@@ -87,7 +87,7 @@ func (s *scanner) Lex(lval *sqlSymType) int {
 
 	case NULLS:
 		switch s.nextTok.id {
-		case FIRST, LAST:
```

Comment:
This is to handle the `NULLS FIRST` and `NULLS LAST` syntax which is a postgres extension to specify where `NULLS` sort. I think we've already deleted that syntax. So this can actually be completely removed. Let me do that in a follow-up PR.